code401challenges/Data-Structures/src/main/java/datastructure/LinkedList.java

```diff
@@ -1,3 +1,152 @@
 package datastructure;
 
+public  class LinkedList {
+    Node head; // Head of the linked list
+
+    LinkedList() {
+        this.head = null;
+    }
+
```

Comment:
good start on getting rid of the Library.java class, you still have your Node class nested though, would you please separate that out into it's own file. 

example:
![Screen Shot 2019-08-13 at 7 18 29 PM](https://user-images.githubusercontent.com/36779764/62990041-37ae7100-bdff-11e9-905e-edddcfc7cedb.png)